drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -336,12 +336,23 @@ def alerts
       load_ar_association(object, :alerts, scope: Hmis::ClientAlert.active).sort_by(&:created_at).reverse
     end
 
+    # not optimized for batch queries, causes n+1 queries
     def hud_chronic
       return unless current_permission?(permission: :can_view_hud_chronic_status, entity: object)
 
-      # causes n+1 queries
-      enrollments = object.enrollments.where(data_source_id: current_user.hmis_data_source_id)
-      !!object.destination_client&.as_warehouse&.hud_chronic?(scope: enrollments)
+      open_enrollments = object.enrollments.open_excluding_wip.preload(:ch_enrollment, :project)
```

Comment:
Can you add a comment noting that we are intentionally not checking enrollment viewability here? can_view_hud_chronic_status grants permission to view HUD Chronic status across all clients enrollments